docs/landing-zone-v2/README.md

```diff
@@ -366,6 +368,53 @@ cd pbmm-landingzone
     Review and customize all packages' `setters.yaml` with the unique configuration of your landing zone.  
     For example "core-landing-zone" will have the same [setters.yaml](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/solutions/landing-zone-v2/setters.yaml) as in the repo in the root of the pkg directory.
 
+### <a name='deploy-the-infrastructure-using-kpt'></a>2b. Deploy the infrastructure using KPT
+This section is an alternative to GitOps in the following section 3.
+
+#### <a name='gatekeeper-policies-kpt'></a>gatekeeper-policies
+Optional
```

Comment:
Could we add something along the lines of strongly recommended or required for Prod?